docs/topics/readme/crud-README.adoc

```diff
@@ -16,13 +16,16 @@ $ oc login -u developer -p developer
 
 $ oc new-project MY_PROJECT_NAME
 
+# Ensure that you use the login credential values specified below when creating the database application.
```

Comment:
Same as above, "login credential values" -> "username, password and database name".